assets/src/dashboard/app/views/editorSettings/publisherLogo/popoverLogoContextMenu.js

```diff
@@ -51,20 +47,12 @@ function PopoverLogoContextMenu({
     [contextMenuId]
   );
 
-  const handleFocusOut = useCallback(() => {
-    if (contextMenuId.value === publisherLogo.id) {
-      onMoreButtonSelected(-1);
-    }
-  }, [publisherLogo.id, contextMenuId.value, onMoreButtonSelected]);
-
-  useFocusOut(
-    popoverMenuContainerRef,
-    contextMenuId.value === publisherLogo.id ? handleFocusOut : noop,
-    [contextMenuId, publisherLogo.id]
-  );
```

Comment:
Removing focus utils. This should reduce the number of focus trap bugs that pop up.